<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Total</class>
 <widget class="QWidget" name="Total">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1338</width>
     <height>830</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	background-color: rgba(240, 240, 240,0.87);
	border-radius: 35px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border-image: url(:/red.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: 500 18pt &quot;Century Gothic&quot;;
	background:transparent;
        color: rgb(65, 65, 65);  
        border-radius: 9px;  
        border-style: outset;
		border: none;
}

QPushButton:hover{
	   color: black;   
        border-radius: 9px; 
        border-style: outset;
		border: none;
		background: rgba(163, 163, 163,0.7);
}</string>
    </property>
    <property name="text">
     <string>      Start     </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: 500 18pt &quot;Century Gothic&quot;;
	background:transparent;
        color: rgb(65, 65, 65);  
        border-radius: 9px;  
        border-style: outset;
		border: none;
}

QPushButton:hover{
	   color: black;   
        border-radius: 9px; 
        border-style: outset;
		border: none;
		background: rgba(163, 163, 163,0.7);
}</string>
    </property>
    <property name="text">
     <string>    Delete   </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_step">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: 500 18pt &quot;Century Gothic&quot;;
	background:transparent;
        color: rgb(65, 65, 65);  
        border-radius: 9px;  
        border-style: outset;
		border: none;
}

QPushButton:hover{
	   color: black;   
        border-radius: 9px; 
        border-style: outset;
		border: none;
		background: rgba(163, 163, 163,0.7);
}</string>
    </property>
    <property name="text">
     <string>      Step    </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_id_t">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_id">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_data_t">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_data">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>81</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_shrink">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border-image: url(:/shrink.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>27</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border-image: url(:/about.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Century Gothic&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Function:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Century Gothic&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Infomation:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_bar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>1057</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(226, 226, 226,0.9);
border:1px solid rgb(216, 216, 216);
border-top-right-radius:35px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid black;
font: 700 14pt &quot;Adobe Caslon Pro Bold&quot;;
color: rgb(0, 85, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{font: 700 16pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;border: none;
border-radius:0px}</string>
     </property>
     <property name="text">
      <string>Please enter the number of nodesï¼š</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_generate">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>38</width>
       <height>38</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;generate now&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
border-radius:0px;
border-image: url(:/generate.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>49</y>
      <width>40</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgb(216, 216, 216);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>50</y>
      <width>38</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgb(216, 216, 216);
border-bottom-right-radius:35px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Coordinates_t">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Coordinates:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Coordinates">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_fluency_t">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string>id&amp;Delay</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_fluency">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_more">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>780</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image: url(:/more1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_flu_t">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Fluency:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_flu">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Adobe Caslon Pro Bold&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
